### DISCUSSION TITLE
Can you remove the sourceMappingURL in js files after upload

### DISCUSSION THREAD

**Discussion by avegancafe** [getsentry/support#108] [Support] [answered]
If I am using the webpack plugin to upload sourcemaps directly to sentry, do I still need to include the sourceMappingURL in all of the js files? Wouldn't that defeat the purpose of specifically uploading the sourcemaps to sentry by having those sourcemaps still publicly available? 

EDIT:
Also, apologies if this is answered anywhere! I couldn't find the specific answer, but feel free to boot me towards some docs or another issue if it's already been answered

**Comment by souredoutlook** [comment, marked as answer]
Hi there, the webpack plugin now supports the `hidden-source-map` devtool, which will remove the sourceMappingURL from your files. You can read about how to set it up [here](https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/#use-hidden-source-map).
